gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,53 @@
+from oslo_config import cfg
```

Comment:
Why are nsf/core files in this pull request?  Assuming these will be the same as in pull request #48